### ISSUE TITLE
labels in image search

### ISSUE THREAD

**Issue by ch-sander** [lombardpress/lombardpress-webjs#101]
I think the image search results should have meaningful (to a human reader) labels, e.g. "figure n (on page x in manifestation y)" instead of the internal ID like "p9i7y5-e21765-Fd1e121". This ID could be embedded somewhere else or as tooltip.

**Comment by jeffreycwitt** [comment]
I think commit 25df3eff1738719d4459eca846614c14fd8d8ff0 makes first step toward resolving this. It maybe not be the perfect or final solution, but I think it works for now. (See gif below)

![imageSearchResults](https://user-images.githubusercontent.com/1146685/185925635-5b276d42-0821-40e7-9bd0-8dd48725e7df.gif)

**Comment by ch-sander** [comment]
Yes, that looks cool! Maybe one could even hide the internal ID. It's to long to remember and cannot be copied to clipboard in the popover anyway.

**Comment by ch-sander** [comment]
And maybe grouping the images by work would (e.g., as sub-headers with forces line break) make the grid even more user friendly.
